internal/types/types.go

```diff
@@ -48,6 +48,7 @@ import (
 	"time"
 )
 
+// MaxDocumentLen is maximum BSON object size.
```

Comment:
```suggestion
// MaxDocumentLen is the maximum BSON object size.
```

I think it is worth checking comments with Grammarly